src/meta/base_values.yaml

```diff
@@ -0,0 +1,514 @@
+# Terrain
+minecraft:stone: 1
+minecraft:deepslate: 2
+minecraft:cobbled_deepslate: 2
+minecraft:calcite: 5
+minecraft:tuff: 2
+minecraft:grass_block: 1
+minecraft:dirt: 1
+minecraft:podzol: 1
+minecraft:rooted_dirt: 1
+minecraft:cobblestone: 1
+minecraft:bedrock: -1
+minecraft:sand: 5
+minecraft:red_sand: 5
+minecraft:gravel: 5
+minecraft:coal_ore: 25
+minecraft:deepslate_coal_ore: 25
+minecraft:iron_ore: 100
+minecraft:deepslate_iron_ore: 100
+minecraft:copper_ore: 50
+minecraft:deepslate_copper_ore: 50
+minecraft:gold_ore: 250
+minecraft:deepslate_gold_ore: 250
+minecraft:redstone_ore: 100
+minecraft:deepslate_redstone_ore: 100
+minecraft:emerald_ore: 1000
+minecraft:deepslate_emerald_ore: 1000
+minecraft:lapis_ore: 100
+minecraft:deepslate_lapis_ore: 100
+minecraft:diamond_ore: -1
+minecraft:deepslate_diamond_ore: -1
+minecraft:pointed_dripstone: 1
+
+# Saplings
+minecraft:oak_sapling: 20
+minecraft:spruce_sapling: 20
+minecraft:birch_sapling: 20
+minecraft:jungle_sapling: 20
+minecraft:acacia_sapling: 20
+minecraft:dark_oak_sapling: 20
+
+# Nether
+minecraft:crimson_nylium: 1
+minecraft:warped_nylium: 1
+minecraft:nether_gold_ore: 100
+minecraft:nether_quartz_ore: 100
+minecraft:ancient_debris: 2000
+minecraft:budding_amethyst: -1
+minecraft:exposed_copper: 400
+minecraft:weathered_copper: 350
+minecraft:oxidized_copper: 300
+minecraft:netherrack: 1
+minecraft:soul_sand: 1
+minecraft:soul_soil: 1
+minecraft:basalt: 1
+minecraft:smooth_basalt: 1
+minecraft:blackstone: 1
+minecraft:crying_obsidian: 200
+minecraft:gilded_blackstone: 1
+minecraft:cracked_polished_blackstone_bricks: 1
+
+# Wood
+minecraft:oak_log: 40
+minecraft:spruce_log: 40
+minecraft:birch_log: 40
+minecraft:jungle_log: 40
+minecraft:acacia_log: 40
+minecraft:dark_oak_log: 40
+minecraft:crimson_stem: 40
+minecraft:warped_stem: 40
+minecraft:stripped_oak_log: 40
+minecraft:stripped_spruce_log: 40
+minecraft:stripped_birch_log: 40
+minecraft:stripped_jungle_log: 40
+minecraft:stripped_acacia_log: 40
+minecraft:stripped_dark_oak_log: 40
+minecraft:stripped_crimson_stem: 40
+minecraft:stripped_warped_stem: 40
```

Comment:
Seems pretty high. Wood is easily obtained, so 40 stone for the same value as a log? That's a bit high